### PR TITLE
feat: Define Helm chart

### PR DESCRIPTION
**Pull request by matteocellucci**

### CODE REVIEW

**Comment by matteocellucci** [comment]
Depends on #4

File: helm/values.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+
+securityContext: {}
+
+service:
+  type: NodePort
```
Comment (diegolagospagopa) [inline helm/values.yaml:24]: This value must be ClusterIp: https://github.com/pagopa/selfcare-ms-product/blob/111c2a7d3516ec581747fead5918727b4b0bc438/manifests/service.yml#L6

File: helm/values.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  port: 80
+
+resources: {}
```
Comment (diegolagospagopa) [inline helm/values.yaml:27]: please add to resources:

```yaml
  requests:
    memory: "64Mi"
    cpu: "250m"
  limits:
    memory: "128Mi"
    cpu: "500m"
```

To help with the best practice

File: helm/values.yaml (hunk 1 above)
Comment (matteocellucci) [reply helm/values.yaml:24]: Yep, do you think we should remove the service at all? I mean, could the service be considered as part of the infrastructure and not of the microservice?

**Comment by diegolagospagopa** [comment]
It's would be great to have an example to how to mount a configMap as volume, so for example have a configuration file for the application.

https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#add-configmap-data-to-a-volume

**Comment by diegolagospagopa** [comment]
It would be great to have a startupProbe in deployment: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes

**Comment by diegolagospagopa** [comment]
it would be great to have 3 differents values.yaml, each for env

File: helm/values.yaml (hunk 1 above)
Comment (diegolagospagopa) [reply helm/values.yaml:24]: No, the service is always part of the project. The ingress can be something shared

**Review by pasqualedevita — APPROVED** [review]

**Pull request merged**